src/main/java/frc/robot/commands/MoveToHeight.java

```diff
@@ -0,0 +1,29 @@
+package frc.robot.commands;
+
+import com.spikes2212.command.genericsubsystem.commands.smartmotorcontrollergenericsubsystem.
+        MoveSmartMotorControllerGenericSubsystem;
+import com.spikes2212.control.FeedForwardSettings;
+import com.spikes2212.control.PIDSettings;
+import com.spikes2212.util.UnifiedControlMode;
+import frc.robot.subsystems.Elevator;
+
+import java.util.function.Supplier;
+
+public class MoveToHeight extends MoveSmartMotorControllerGenericSubsystem {
+
+    private final Elevator elevator;
+    public MoveToHeight(Elevator elevator, PIDSettings pidSettings, FeedForwardSettings feedForwardSettings,
+                        Supplier<Double> setpoint) {
+        super(elevator, pidSettings, feedForwardSettings, UnifiedControlMode.POSITION, setpoint);
+        this.elevator = elevator;
+    }
+
+    @Override
+    public boolean isFinished() {
+        if (elevator.isMin() || elevator.isMax()) {
```

Comment:
if the elevator is currently closed, the bottom limit is pressed. when i want to move up, the isFinished will return true, even though i CAN and SHOULD go up. when going up, you can ignore isMin, and when going down, you can ignore isMax.